src/backend/distributed/citus--7.0-4--7.0-5.sql

```diff
@@ -95,4 +95,21 @@ COMMENT ON FUNCTION master_add_inactive_node(nodename text,nodeport integer,
                                              groupid integer, noderole noderole)
   IS 'prepare node by adding it to pg_dist_node';
 
+DROP FUNCTION master_activate_node(text, integer);
+CREATE FUNCTION master_activate_node(nodename text,
```

Comment:
We usually don't modify existing SQL files, since some users build from `master` and missing a change would force them to drop the extension. Probably doesn't affect anyone, but may better to add a new SQL file?